numpy/ma/tests/test_core.py

```diff
@@ -2581,6 +2581,14 @@ def test_no_masked_nan_warnings(self):
             # also check that allclose uses ma ufuncs, to avoid warning
             allclose(m, 0.5)
 
+    def test_masked_array_underflow(self):
+        with(np.errstate(under="raise")):
+            x = np.arange(0, 3, 0.1)
```

Comment:
Maybe move this line and the next two or of the with context, as the test is about the masked array.

Reply:
Ok i'll do so